app/serializers/export/post_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Export
+  class PostSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
+               :public,
+               :diaspora_handle,
+               :type,
+               :image_url,
+               :image_height,
+               :image_width,
+               :likes_count,
+               :comments_count,
+               :reshares_count,
```

Comment:
Doesn't it contain the current (when exported) count though? IMHO that is relevant information for the user :)